platform/sun50i/devices.c

```diff
@@ -23,6 +25,13 @@ static struct device ccu = {
 	.regs = DEV_CCU,
 };
 
+static struct device pio = {
```

Comment:
Needs to match `r_pio` from above.

Also needs `clockdev` and `clock`

Reply:
Fixed